### ISSUE TITLE
[Bug]: Android app will not play an audiobook or podcast when it can't find the cover/metadata files

### ISSUE THREAD

**Issue by cameron98** [advplyr/audiobookshelf-app#1340]
### What was the Problem?

When the paths in the audiobookshelf database to the metadata/covers were pointing to the wrong location, the audiobookshelf app was unable to play any audio files even though the web interface and shelfplayer apps were still able to. When attempting to play the audio file, an red error box popped up to say stream failed. During troubleshooting of the issue, I tried clearing the caches and rescanning all libraries. I also cleared all data and cache from the mobile app, tried reinstalling the app and removing and re-adding the server to the app. 

### Steps to Reproduce the Issue

The issue was encountered when attempting to migrate my audiobookshelf instance from a RHEL system to a docker container where the path changes from `/var/lib/audiobookshelf/metadata` to just `/metadata`. I think the issue could also be reproduced by renaming the metadata folder. 

### What was Expected?

For the app to play audio, even if it was unable to find a cover to send to the client or any other related metadata. 

### Phone Model

Pixel 7a

### Phone OS

Android 14

### Audiobookshelf App Version

Android App - 0.9.76

### Installation Source

Google Play Store

### Additional Notes

I understand that the use case I mentioned when encountering the bug is likely not supported, however I considered it worth raising as the issue may also make the system unusable for mobile users in the case of those files being corrupted/accidentally deleted. At least in the other cases I tested such as the shelfplayer app and the web interface, the books could still be listened to, even with no cover art etc. I was able to resolve the issue by dumping the sqlite database and doing a sed replace on the filepaths to their new location, then restoring the database back again, so I am confident that this was the cause of the issue.

**Comment by ISO-B** [comment]
I couldn't reproduce this. I tried following things:
1. Removed cover file for single book from metadata/items/item_id/ folder.
  - App and webui both showed invalid cover image
2. Removed metadata from same folder.
  - Everything continued to work as before
3. Removed cached things related to book
  - Everything continued to work as before.
  
Also booted server and app on phone after each step. I was able to play audiobook on app and browser.

Tested with:
 - Android app 0.9.74, Server v2.14.2 and v2.16.0, OnePlus One Android 11
 - Android app 0.9.76, Server v2.16.0, OnePlus 8T Android 14
